XRTK-Core/Packages/com.xrtk.core/Definitions/BaseMixedRealityExtensionDataProviderProfile.cs

```diff
@@ -1,11 +1,9 @@
 ﻿// Copyright (c) XRTK. All rights reserved.
 // Licensed under the MIT License. See LICENSE in the project root for license information.
 
-using XRTK.Interfaces;
-
 namespace XRTK.Definitions
 {
-    public abstract class BaseMixedRealityExtensionDataProviderProfile : BaseMixedRealityServiceProfile<IMixedRealityDataProvider>
+    public abstract class BaseMixedRealityExtensionDataProviderProfile : BaseMixedRealityProfile
```

Comment:
Note: `IMixedRealityExtenstionDataProviders` should not have additional configurations registered under them.

(Although, in theory we could, this helps prevent issues around recursive registration of services and other data providers. I think it'll be easier to understand and manage if this list was flat)